employees.rb

```diff
@@ -0,0 +1,15 @@
+class Employee
+  attr_accessor :name, :department, :review, :satisfactory, :salary, :give_raise
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @department = nil
+    @review = nil
+    @satisfactory = true
+  end
+
+  def give_raise
+    @salary + 1000
```

Comment:
A fixed raise is okay, but we were shooting for having a raise method which would allow you to pass in a number or a percentage to use for the raise.